server/db/models/Stretch.js

```diff
@@ -39,4 +40,43 @@ const Stretch = db.define('stretch', {
   }
 })
 
+Stretch.getStretchesByCohort = async function(cohortId) {
+  const cohortStretches = await CohortStretch.findAll({
```

Comment:
similar refactoring to User method:

Stretch.findAll({
include: [
{
attributes: [],
model: CohortStretch,
where: {
cohortId
}
}
]
})
.then(students => res.json(students))
.catch(next)

However, I think this is different in the sense we actually very much want the information in the CohortStrecth model so it would actually be:

Stretch.findAll({
include: [
{
model: CohortStretch,
where: {
cohortId
}
}
]
})
.then(students => res.json(students))
.catch(next)